_overviews/core/collections-migration-213.md

```diff
@@ -0,0 +1,330 @@
+---
+layout: singlepage-overview
+title: Migrating a Project to Scala 2.13's Collections
+permalink: /overviews/core/:title.html
+---
+
+This document describes the main changes for collection users that migrate to Scala 2.13 and shows
+how to cross-build projects with Scala 2.11 / 2.12 and 2.13.
+
+For an in-depth overview of the Scala 2.13 collections library, see the [collections guide]({{ site.baseurl }}/overviews/collections-2.13/introduction.html). The implementation details of the 2.13 collections are explained in the document [the architecture of Scala collections]({{ site.baseurl }}/overviews/core/architecture-of-scala-213-collections.html).
+
+## What are the major changes?
+
+See the list at the top of the [2.13.0-M4 release notes](https://github.com/scala/scala/releases/tag/v2.13.0-M4).
+
+## What are the breaking changes?
+
+The following table summarizes the breaking changes:
+
+| Description | Old Code | New Code | Automatic Migration Rule |
```

Comment:
Since the table does not render well and it only has few entries, we could just replace table rows with paragraphs:

~~~ markdown
### Operation `to[C[_]]`

Operation `to[C[_]]` has been removed.

Scala 2.12 code:

```scala
xs.to[List]
```

Scala 2.13 code:

```scala
xs.to(List)
```

Automatic migration rules: `Collection213Upgrade` and `Collections213CrossCompat`
~~~

Reply:
i left the tables for now. not sure what's the best way to fix it... is there something like a scroll-div we can use? for code blocks that is done automatically.